### PR TITLE
Converted projects to Netstandard 2.0 (Net 4.0 still supported)

### PR DESCRIPTION
**Pull request by dulfe**
Just project changes so netstandard 2.0 is supported. Actual code has not been changed (Everything was compatible).

I noticed a couple of forks that also support netstandard 2.0 but they are out of date. This uses the latest commit (as on January 2018)

### CODE REVIEW

**Comment by dulfe** [comment]
Well.. travis failed because there are some issues with the Makefile.am file... I have no idea what that is or how that works... so I will try and research how to locally test those files.

**Comment by decriptor** [comment]
Looking at it for 5 seconds, it could be that the changes you made aren't compatible with xbuild (notice this isn't msbuild). Also, any reason you removed the AssemblyInfo.cs?

**Comment by dulfe** [comment]
Hello @decriptor ... I just found out what you just told me :)... xbuild is been used .

I checked online and seems like xbuild has issues with netstandard 2.0 (https://github.com/dotnet/core/issues/862).... and it is been deprecated?... I will have to look into it :)

I removed the AssemblyInfo.cs file because when I compiled using the new project format it gave me exceptions (duplicated attributes)... I meant to put all that information in the project file but I forgot... I apologize.... if I am able to make xbuild work with the new project format I will put that info in there.

Thank you!

**Comment by dulfe** [comment]
@decriptor I actually already put the AssemblyInfo information into the Project file... so all the attributions are in there. :)

**Comment by dulfe** [comment]
Ok... I was not able to compile it using XBUILD... and MSBUILD only worked after installing `dotnet-sdk-2.1.4` and `mono-complete` and then restoring packages using `nugget restore` and `dotnet restore` (yes, 2 restores) 

Well... I guess that shows that I normally don't do cross-platform compiling. :)

I will have to learn more about Linux before sending a new pull request... sorry for wasting your time.

PS: I also had to fix a stranded file entry in the test.csproj file.

**Comment by decriptor** [comment]
Can you rebase this onto master? I just merged a PR to switch travis-ci over to msbuild.

**Comment by dulfe** [comment]
@decriptor It failed again, but this time it had issues restoring packages (Timeouts).

**Comment by decriptor** [comment]
@dulfe I restarted the build because the timeouts were bogus. There are different errors now

**Comment by dulfe** [comment]
@decriptor Now it works... I modified the travis file so it enables dotnet support.... I also merged the latest commit to master.

**Comment by decriptor** [comment]
@dulfe awesome! I want to leave this here for a little bit to get more people looking at it before merging this. I think there'll be a lot of very happy people to see this.

**Comment by FasettoAndrew** [comment]
Any idea when this will get merged? Would be good to remove my temp .net standard Nuget package and use the official taglib sharp package :)

File: src/taglib-sharp.csproj [hunk 1]
```diff
@@ -1,297 +1,27 @@
...
-    <CheckForOverflowUnderflow>True</CheckForOverflowUnderflow>
-    <AssemblyOriginatorKeyFile>..\taglib-sharp.snk</AssemblyOriginatorKeyFile>
+    <RootNamespace>TagLib</RootNamespace>
     <SignAssembly>true</SignAssembly>
-    <DocumentationFile>taglib-sharp.XML</DocumentationFile>
+    <AssemblyOriginatorKeyFile>..\taglib-sharp.snk</AssemblyOriginatorKeyFile>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net4|AnyCPU'">
+    <DocumentationFile>bin\Debug\net4\taglib-sharp.xml</DocumentationFile>
```
Comment (decriptor) [inline src/taglib-sharp.csproj]: this should be `net40`.
```sshaw@shawmsxamthree:~/Projects/taglib-sharp/src (dulfe-master)$ cd bin/Debug/net
net40/          netstandard2.0/```

**Comment by decriptor** [comment]
Since the `taglib-sharp.XML` is generated on each build we probably shouldn't have this committed in the source tree. Maybe remove that and update the .gitignore file?

**Comment by decriptor** [comment]
I fixed this and manually merged it.

**Comment by JoshSchreuder** [comment]
@decriptor would you be able to publish an updated Nuget package with all these changes? I'm mainly interested in getting a netstandard compatible build through Nuget for one of my project. Thanks in advance!

**Pull request merged**